src/main/java/io/confluent/idesidecar/restapi/exceptions/ExceptionMappers.java

```diff
@@ -68,15 +68,57 @@ public Response mapTemplateNotFoundException(
         .build();
   }
 
+  @ServerExceptionMapper
+  public Response mapFlagNotFoundException(
+      FlagNotFoundException exception
+  ) {
+
+    Failure failure = new Failure(
+        exception,
+        Status.NOT_FOUND,
+        "resource_missing", exception.getMessage(), uuidFactory.getRandomUuid(),
+        null);
+    return Response
+        .status(Status.NOT_FOUND)
+        .entity(failure)
+        // Explicitly set the content type to JSON here
+        // since the resource method may have set it to something else
+        .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON)
+        .build();
+  }
+
+  @ServerExceptionMapper
+  public Response mapFeatureFlagFailureException(
+      FeatureFlagFailureException exception
+  ) {
+
+    Failure failure = new Failure(
+        exception,
+        Status.INTERNAL_SERVER_ERROR,
+        Status.INTERNAL_SERVER_ERROR.getReasonPhrase(),
+        Status.INTERNAL_SERVER_ERROR.getReasonPhrase(),
+        uuidFactory.getRandomUuid(),
+        null
+    );
+    return Response
+        .status(Status.NOT_FOUND)
```

Comment:
This only happens if the sidecar is unable to parse the LD response, so I would consider it an internal error. But as you suggested, I will add 500 to the OpenAPI spec.

Reply:
Made this change.

Reply:
Using `Status.INTERNAL_SERVER_ERROR` makes a lot of sense here. Can you please also update the `Response.status` in the exception mapper function?

Reply:
Actually, the `FeatureFlagFailureException` are logged, since that means we had trouble parsing the LD response, but that parsing and this error only occur on background threads and does not occur on REST API request threads. So I'm going to remove this exception mapping.